###  Quantitative genetics of adaptation

\documentclass[11pt]{article}

%% Language and font encodings
\usepackage[english]{babel}
\usepackage[utf8x]{inputenc}
\usepackage[T1]{fontenc}

%% Sets page size and margins
\usepackage[a4paper,top=2.5cm,bottom=2.5cm,left=2.5cm,right=2.5cm,marginparwidth=1.75cm]{geometry}

%% Useful packages
\usepackage{amsmath,amssymb}
\usepackage{graphicx}
\usepackage[colorinlistoftodos]{todonotes}
\usepackage[colorlinks=true, allcolors=blue]{hyperref}
\usepackage{color}

\definecolor{jriblue}{rgb}{0.6, 0.8, 0.9}

\newcommand{\cj}[1]{\todo[size=\scriptsize, color=green]{#1}}
\newcommand{\jri}[1]{\todo[size=\scriptsize, color=jriblue]{#1}}
\newcommand{\dr}[1]{\todo[size=\scriptsize, color=orange]{#1}}
\newcommand{\citex}[1]{\textbf{\textcolor{red}{CITE(#1)}}}
\newcommand{\X}{\textbf{\textcolor{red}{X}}}

\title{}

\begin{document}
\textbf{Quantitative Genetics of Adaptation}

While Mendel's initial genetic work focused on traits with discrete variation, most plant phenotypes --- from height to flowering time to yield --- are quantitative, showing continuous, non-discrete variation and are usually governed by a large number of  loci.
Genome wise association (GWA) studies use genotype markers to identify the genetics underlying quantitative traits by associating trait variation with a region of the genome called a quantitative trait locus (QTL).
These studies have been successful at identifying ... \jri{cite a few examples}
Nonetheless, GWA face a number of challenges in correctly identifying small genetic signals in complicated phenotypic variation. 
There are many factors that can confound the results of a GWA study, including sample size, environmental interactions, population structure, and statistical biases that favor intermediate allele frequencies or overestimating QTL effect sizes \cite{josephs2017can}. 
As a result, the QTL identified rarely account for most of the phenotype trait variation observed \cite{huang2010genetic}, and we still know very little about the number of loci and effect size of quantitative traits in plants.

Developing methods to better characterize quantitative traits in plants is important for identifying genes underlying traits of interest, breeding improved crops, and understanding how phenotypes evolve in nature. 
The overall goal of this project is to combine different data types and different but established methods for discovering QTL to better characterize the genetic architecture underlying quantitative traits.
Here, I propose to use gene expression as a model quantitative trait to characterize the genetic architecture of quantitative traits in \textit{Zea mays} populations for an improved outcome of QTL discovery.  
I also aim to use the results of this approach to measure selection on quantitative traits in both maize and in teosinte populations.

A common approach is to use physiological phenotypes, such as height, for the traits in a GWA study.
Another approach is to use gene expression data as the trait in a GWA and associated genetic markers are called expression QTL (eQTL).
This can show cis and trans regulation of the expression of a gene, as it can uncover a QTL in or near the gene region (cis-acting), or a QTL far away from the gene that has an impact on gene expression (trans-acting). 
A potentially powerful use of gene expression lies in it's ability to be clustered by co-regulation patterns.
I clustered genes by their expression patterns and generate a representative gene expression value of that cluster as a trait.
That representative gene value for the cluster can then be used the trait in a GWA, and any associated QTL are possible cis or trans regulators of the known genes within the cluster.
Using gene expression as a trait in GWA is a so far known but under-explored method to identify QTL.

In this project, I aim to use publicly available data, and data generated by projects within the Ross-Ibarra and Runcie labs to meet the following aims:

\textbf{1.} What is the genetic architecture underlying phenotypic traits and RNA co-expression clusters?

\textbf{2.} How well do each of these approaches explain the variance of their quantitative traits? \jri{i like combining these two}

\textbf{3.} Is there evidence of local adaptation of these quantitative traits among the populations samples?

\textbf{4.}  Can this approach be used to identify quantitative traits that have been under selection in teosinte populations? \jri{i would combine 3 and 4 too}

To address \textbf{aim 1}, I will look for significant QTL for agronomic traits using phenotype and expression data and gene expression data collected by \cite{hirsch2014insights} (Figure \ref{manhattan}) and genotype marker data by \cite{romay2013comprehensive}.
In my preliminary analyses I have analyzed 13 different agronomic traits from 292 maize lines. 
I used GEMMA \cite{zhou2012genome} to identify QTL for each trait and calculate the effect sizes for each marker using a linear mixed model that took population structure into account.

To cluster genes by co-expression for the \textbf{aim 1} GWA on gene clusters, we used the R package WGCNA (weighted correlation network analysis) \cite{langfelder2008wgcna} and gene expression data for 15,205 genes \cite{hirsch2014insights}.
The network analysis generated 15 gene modules that had a range of 30 genes to 7934 genes per cluster.
WGCNA generates a representative gene for all the genes in a cluster, and the values of this "gene" are used in the GWA. 
One gene cluster (labeled the "black" gene cluster) had a clear peak on chromosome 3 (Figure \ref{manhattan}B).
However, the gene located within this peak, the floral transcription factor \textit{zag2}, is not one of the 414 genes in the black gene cluster, indicating a likely example of trans regulation.  \jri{yes move this before the phenotype results}

The results of the GWA for the phenotypic traits had a total of 130 significant QTL (p-value < 1e-4) across all 13 traits, and no clear peaks indicating large effect QTL (Figure \ref{manhattan}A). 
This may be due to the highly quantitative nature of the traits or due to some confounding feature of the GWA.

\begin{figure}
\includegraphics[width=0.5\linewidth]{figures/stalkman.pdf}
\includegraphics[width=0.5\linewidth]{figures/black_marked.pdf}
\caption{Manhattan plots showing GWA results for \textbf{A.} stalk diameter and \textbf{B.} the black expression module. Green dots in \textbf{B} indicate the location of genetic markers in genes included in the Black cluster.}
\label{manhattan}
\end{figure}

To assess the success of the GWA approaches in \textbf{aim 1}, I will then use modeling approaches \cite{yang2011gcta} to estimate how much of the genetic variance for phenotypes and expression can be explained by the significant markers from GWA.

\jri{another sentence or so about what you will do with these results. what if they suck? or perhaps instead a sentence or two about how expression clustering lets you evaluate these approaches because you *know* the underlying loci}

After characterizing the genetic architecture of phenotypes and gene expression, I propose in \textbf{aim 3}, to search for patterns of selection on quantitative traits using the polygenic \cj{can I say quantitative here? to keep it simple and reduce jargon?} approach described in \cite{berg2014population}. 
The $Q_x$ statistic proposed by \cite{berg2014population} measures if there is more variability in the genetic values among populations than can be explained by drift or relatedness, which would indicate local adaptation on that trait.  \jri{nice explanation}
I will use the top 200 QTL from my GWA results to calculate $Q_x$ and evaluate covariance between QTL allele frequency and the direction of the QTL effects. \jri{on what samples? in what populations?}

For \textbf{aim 4}, in addition to the inbred maize lines from the phenotype dataset \cite{hirsch2014insights}, the Ross-Ibarra lab has access to teosinte population genotype and phenotype data.
If this approach proves useful for the inbred maize lines, it might prove useful to show local adaptation of quantitative traits in teosinte populations.
Maize and teosinte provide well characterized and well studied models to answer this questions, as there is an abundance of data and diverse populations that have been subjected to various environmental conditions, both human and natural.  \jri{i'd end with a general statement about how this project will demonstrate how quant. gen. approaches can use data from mapping populations -- which are abundant for crops -- to learn about the underlying genetics of phenotypes of interest as well as study selection on these phenotypes in natural and agricultural settings}


\bibliographystyle{naturemag}
\bibliography{sample}

\end{document}


#########
Citations

1. Berg, J. J. & Coop, G. A Population Genetic Signal of Polygenic Adaptation. PLOS Genetics 10, e1004412 (2014).
2. Langfelder, P. & Horvath, S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics 9, 559 (2008).
3. Josephs, E. B., Stinchcombe, J. R. & Wright, S. I. What can genome-wide association studies tell us about the evolutionary forces maintaining genetic variation for quantitative traits? New Phytol 214, 21–33 (2017).
4. Ross-Ibarra, J., Morrell, P. L. & Gaut, B. S. Plant domestication, a unique opportunity to identify the genetic basis of adaptation. PNAS 104, 8641–8648 (2007).
5. Zhou, X., Stephens, M. Genome-wide efficient mixed-model analysis for association studies. Nature Genetics 44, 821-824 (2012). 
6. Hill, W. G. Understanding and using quantitative genetic variation. Philosophical Transactions of the Royal Society of London B: Biological Sciences 365, 73–85 (2010).
7. Huang, Y.-F. et al. The Genetic Architecture of Grain Yield and Related Traits in Zea maize L. Revealed by Comparing Intermated and Conventional Populations. Genetics 186, 395–404 (2010).
8. Romay, M. C. et al. Comprehensive genotyping of the USA national maize inbred seed bank. Genome Biology 14, R55 (2013).






