#########
Introduction
Relevance and Justification

Many traits in plants that are important to both humans and to plant populations are quantitative traits, i.e. they have multiple, sometimes are large number of genes controlling that trait. Plant traits such as yield, height, and stem thickness, are quantitative, and have been under selective pressure from both humans and their environment. During the domestication process, humans have altered the genetic architecture of crop plants in ways analogous to local polygenic adaptation: small allele frequency shifts at many different loci. Certain large effect loci have been associated with the domestication process, such as teosinte branching 1 (tb1). However, tb1 has been shown to only explain X, sometimes as small as Y%, of the phenotype [citation]. This is true for other plant QTL, indicating there is much the plant science community does not know about quantitative traits and their genetic architecture.
A widely used approach to gene discovery or analysis for quantitative traits is a top-down approach (Ross-Ibarra et al. 2007). This method begins with phenotype data, and associates possibly causative regions of the genome to those traits, using genome wide association (GWA) methods. A list of candidate genes can be generated from these regions, known as quantitative trait loci (QTL). Here, we plan to use phenotypic data and expression data to generate candidate QTL underlying quantitative traits, and use any candidate genes and their effect sizes to look for patterns of local adaptation among maize populations.
Detecting all contributing genes to the quantitative trait is difficult. Confounding factors of a GWAS analysis include environmental impacts, sample size, and quality of genetic information, amount of SNPs used. In addition, there are several statistical challenges to identifying causal QTL. In a GWAS, there is more power to detect common alleles, and alleles with large effect sizes, and this will cause an “overestimation of the average effect size and frequency of GWAS-associated alleles” (Josephs et al. 2017). In addition, there can be an excess in false positives from low-frequency alleles if the original phenotype data is skewed (Josephs et al. 2017). A limited sample size can underestimate the number of QTLs affecting a trait due to multiple loci being misidentified as a single QTL with a large effect (Josephs et al. 2017).
It is important to develop methods to accurately detect small-effect QTL. Discovering the loci underlying plant traits is important to both breeders and crop breeding programs, and to further elucidating plant genetic networks, plant physiology, and plant evolution.
In this project, I aim to leverage publicly available data, and data generated by projects within the Ross-Ibarra to investigate methods of reducing genetic information to facilitate the characterization the genetic architecture of traits under selection using a top-down approach. The

#########
Goals and Objectives


1. What is the genetic architecture underlying agronomic traits measured by Hirsch et al?
2. What is the genetic architecture underlying the gene co-expression modules?
   2.a. How similar are the module eigengenes GWAS with the eQTL of genes within the module?
3. Is there evidence of local adaptation of these quantitative traits among the populations samples?
4. Can we use this approach in other populations? Can I use this approach in teosinte populations that have been subjected to strong environmental selection on a regional scale?

#########
Project Outline

Starting with gene expression and phenotype data publically available from a maize study performed by Hirsch et al (2015?), and with genome by sequencing data from The Maize Diversity Project (Panzea, citation), the original dataset was filtered based on gene expression and percent missing data. The final analyses were performed with 292 lines of maize (Figure 1), 15,206 genes, and 133,943 SNPs. [Can I have some measure of coverage in there, or something about quality of the SNPs, citing panzea paper?] The 292 lines of maize represent 14 different maize families (Figure 1).
We used the WGCNA (weighted correlation network analysis) R package to generate correlated gene clusters based on expression patterns [3]. The network analysis generated 15 gene modules that had a range of 30 genes to 7934 genes per module. WGCNA generates the module eigengene, which is the first principal component for that module - or can be thought of as the expression levels that best represents all the genes in that module.
We used GEMMA (genome-wide efficient mixed-model association) [5] to run the the GWA analyses, with the allowed missing frequency increased to 10% (compared to the standard 5%). The model for the GWA used was:
$$y = x\beta + u + \epsilon$$
Where y is the n-vector of traits, x is the nxp matrix of marker genotypes, \beta is the effect size of the marker. $u$ is the multivariate normal distribution vector of random effects, which includes the a relatedness matrix to correct for population structure in the analysis. The error, \epsilon, is also a multivariate normal distribution.


#########
Project Progress


First, I looked for QTL associated with trait values. There were few significant QTL associated to the traits, though there were X SNPs that were significant (Figure 2), but these may have been false positives. 

2.a. WGCNA Modules
    - about the modules:
        - 13 modules, with the range of Y-Z genes in each module
    - GWAS on the eigengenes of the modules
        - eigengene is the first principal component of the expression data of the genes within the module

Next, I used GEMMA to calculate the genetic associations between the module eigengenes and the population genetic markers. These again produced few significant peaks, though the Black module (383 genes) had a significant and clear peak in chromosome 3 (Figure 2). This peak contained the floral transcription factor zag2. However, zag2 is not included in the Black module, indicating possible trans-regulation by a gene in the Black module. Most of the significant SNPs in the 

2.b. eQTL on expression data of genes
    - 48,136 number of genes [this is not the right number, I thought it was 15K]
    - comparing genes within module to the module eigengene GWAS results (Figure 1)
    - cis and trans effects of SNPs on module eigengenes

Figure 1: this figure includes results from Aim 2.
    a module GWAS results (midnight blue module looked good), with some genes from within the module and their QTL

3. Next step, Qx statistic [Berg and Coop]
    - use the top QTL from a GWAS to look for adaptation among populations for that trait, or on those genes.
    - am I using QTL from WGCNA or from trait GWAS?

The next step is to collect the top SNPs (top 200 SNPs) and to use the Q_x statistic described by Berg and Coop [1] to look for signatures of selection on polygenic traits.

#########
Next Steps and Future Plans

Measures of selection
    - from the GWAS on traits (or on modules), select the top X SNPs from the GWAS
    - get the direction of the effect
    - calculate Qx

Plans for how this can incorporate with the highland project
    - using data gathered from the maize highland project,
        look for maize selection to stressful environmental conditions

Characterize the usefulness of this approach
    - is it accurate?
    - what is the possible false-positive rate [why does Qx not create false positives again?]



#########
Citations
Berg, J. J. & Coop, G. A Population Genetic Signal of Polygenic Adaptation. PLOS Genetics 10, e1004412 (2014).
Langfelder, P. & Horvath, S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics 9, 559 (2008).
Josephs, E. B., Stinchcombe, J. R. & Wright, S. I. What can genome-wide association studies tell us about the evolutionary forces maintaining genetic variation for quantitative traits? New Phytol 214, 21–33 (2017).
Ross-Ibarra, J., Morrell, P. L. & Gaut, B. S. Plant domestication, a unique opportunity to identify the genetic basis of adaptation. PNAS 104, 8641–8648 (2007).
Zhou, X., Stephens, M. Genome-wide efficient mixed-model analysis for association studies. Nature Genetics 44, 821-824 (2012). 



